lib/gui/app.js

```diff
@@ -297,10 +292,9 @@ app.controller('HeaderController', function(OSOpenExternalService) {
     const supportUrl = selectionState.getImageSupportUrl() || DEFAULT_SUPPORT_URL;
     OSOpenExternalService.open(supportUrl);
   };
-
 });
 
-app.controller('StateController', function($rootScope, $scope) {
+app.controller('StateController', function ($rootScope, $scope) {
```

Comment:
Out of curiosity (this may well be another one of my stupid questions) how come this is still using the `function` keyword rather than fat-arrow notation?